doc/user/content/sql/create-sink.md

```diff
@@ -207,6 +198,26 @@ JOIN mz_kafka_sinks USING (id);
  u6        | materialize.public.frank_quotes_sink | frank-quotes
 ```
 
+#### Sizing a sink
+
+To provision a specific amount of CPU and memory to a sink on creation, use the `SIZE` option:
+
+```sql
+CREATE SINK snk FROM src
+INTO KAFKA CONNECTION kafka_connection (TOPIC 'snk')
+FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_connection
+ENVELOPE DEBEZIUM
+WITH (SIZE = 'xsmall');
```

Comment:
```suggestion
CREATE SINK snk FROM src
  INTO KAFKA CONNECTION kafka_connection (TOPIC 'snk')
  FORMAT AVRO USING CONFLUENT SCHEMA REGISTRY CONNECTION csr_connection
  ENVELOPE DEBEZIUM
  WITH (SIZE = 'xsmall');
```

Reply:
Mind if I leave this one as is for now? It's consistent with all the rest of the `CREATE SINK` examples in this doc AFAICT. No objection to reformatting them all though!

Reply:
Sure! Was just looking for revenge from yesterday's PR, but then noticed there's no consistency for sinks.